src/l500/ac-trigger.cpp

```diff
@@ -187,8 +187,16 @@ namespace ivcam2 {
                 if( ! is_auto_trigger_possible() )
                     throw invalid_value_exception( "auto trigger is disabled in the environment" );
                 super::set( value );
-                if( _autocal->_dev.get_depth_sensor().is_streaming() )
-                    _autocal->start();
+                try
+                {
+                    if (_autocal->_dev.get_depth_sensor().is_streaming())
+                        _autocal->start();
+                }
+                catch (std::exception const & e)
+                {
+                    AC_LOG(ERROR, "EXCEPTION caught in access to device: " << e.what());
+                    return;
```

Comment:
This means that start() never happened... so we failed to enable auto trigger, yet the option value has still changed.

I suggest we move line 189 before line 194, into the try block after the if.

Reply:
Done